index.html

```diff
@@ -550,6 +550,138 @@
 template: inverse
 
 ## Serwer danych **Django**
+---
+.left-column[
+  ## Django
+  ### Co robi
+]
+.right-column[
+
+** Co robi serwer? **
+
+* Udostępnia wygenerowany na podstawie modeli interfejs [admina](https://eaterslab.herokuapp.com/admin/).
+
+* Udostępnia automatycznie wygenerowany na podstawie kodu dokument [API](https://eaterslab.herokuapp.com/doc).
+
+]
+
+???
+
+W interfejsie admina mamy dostęp do różnych elementów naszej bazy danych, takich jak 
+użytkownicy, kamery, stołówki ...
+
+API jest generowane automatycznie jednym z zainstalowanych dodatków do Django REST framework.
+
+---
+
+.left-column[
+  ## Django
+  ### Co robi
+  ### API
+]
+.right-column[
+
+** [API](https://eaterslab.herokuapp.com/doc) &ndash; między innymi: **
+
+* Odbiór przez aplikacje mobilne przeróżnych danych &ndash; przede wszystkim dane o stołówkach.
+
+* Wysyłanie przez kamery danych o zajętości stołówki.
+
+]
+
+???
+
+Pobawić się na /doc odbieraniem danych o przykładowych stołówkach i wysłać losowy camera event.
+
+---
+
+.left-column[
+  ## Django
+  ### Co robi
+  ### API
+  ### Baza Danych
+]
+.right-column[
+
+** Baza danych: PostgreSQL **
+
+* Szybka i rozbudowana.
```

Comment:
why fast and furious man? try to explain why you write about these properties (maybe it's why you like it, maybe why you use it, not sure here)